### PR TITLE
Added instructions for meeseeksdev to PR template

### PR DESCRIPTION
**Pull request by hmaarrfk**
Formatted to simplify copy pasting with tripple-clicking

Please squash-merge.

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Comment by stefanv** [comment]
No problem with that: I was uncertain how that would render inside of a list (or, rather, whether it would render inside of a list).

**Comment by hmaarrfk** [comment]
How tripple backticks render. They don't really 

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Considering backporting the PR with 
```
@meeseeksdev backport to v0.14.x
```
- [ ] Next list item

**Comment by hmaarrfk** [comment]
You are right, it won't reorder correctly.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -25,3 +25,4 @@
 - [ ] Check that new functions are imported in corresponding `__init__.py`.
 - [ ] Check that new features, API changes, and deprecations are mentioned in
       `doc/release/release_dev.rst`.
+- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`
```
Comment (soupault) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd change `v0.14.x` to something like `{branch_name}`

Comment (hmaarrfk) [reply .github/PULL_REQUEST_TEMPLATE.md]: We are only supporting v0.14.x right now. This is really meant to make it super easy for you guys to copy paste the command when you click "merge".

**Comment by soupault** [comment]
You are probably right. Updating the template once/release is nothing compared to regular modification of the command.

**Pull request merged**